openapi/client-generator.sh

```diff
@@ -55,9 +56,9 @@ kubeclient::generator::generate_client() {
 
     echo "--- Building docker image..."
     if [ "${USERNAME}" != "kubernetes" ]; then
-        image_name="kubernetes-${USERNAME}-${CLIENT_LANGUAGE}-client-gen:v1"
+        image_name="${USERNAME}-${REPOSITORY}-${CLIENT_LANGUAGE}-client-gen:v1"
```

Comment:
This is reversed from where it was, should be:

`${REPOSITORY}-${USERNAME}-${CLIENT_LANGUAGE}-client-gen:v1`